### PR TITLE
Updating root certificates needed for chocolatey to work.

### PR DESCRIPTION
**Pull request by splatteredbits**

### CODE REVIEW

**Comment by ferventcoder** [comment]
You have two merge commits in here. I know it is just documentation, but can you remove those?

And because it is all the same change I'd be good with a squash (although definitely not required for documentation).

File: Proxy-Settings-for-Chocolatey.md [hunk 1]
```diff
@@ -73,12 +73,10 @@ You've tried everything and Chocolatey still won't install -
...
 
-* Go Daddy Secure Certificate Authority - G2
+To access [chocolatey.org](https://chocolatey.org):
+* [Go Daddy Secure Certificate Authority - G2](https://certs.godaddy.com/repository) (File: `gdroot-g2.crt`; Thumbprint: `47BEABC922EAE80E78783462A79F45C254FDE68B`). (On the download page, this certificate is named *GoDaddy Class 2 Certification Authority Root Certificate - G2* and its listed thumbprint (`45140B3247EB9CC8C5B4F0D7B53091F73292089E6E5A63E2749DD3ACA9198EDA`) is wrong.
 
-![Go Daddy Secure Certificate Authority - G2](https://cloud.githubusercontent.com/assets/63502/13706493/ea4d0aac-e76a-11e5-913e-590dfe775fc1.png)
-
-* COMODO ECC Domain Validation Secure Server CA 2
-
-![image](https://cloud.githubusercontent.com/assets/63502/13706546/1a1d237a-e76b-11e5-9263-e586cef5f744.png)
```
Comment (ferventcoder) [inline Proxy-Settings-for-Chocolatey.md]: So I'm assuming it was intentional that you removed the images?

**Comment by splatteredbits** [comment]
These changes have already been pushed. I don't think I can remove or squash them. I'm a Git newbie, though. Can I modify history that's been pushed? I use Mercurial day-to-day, and modifying history there doesn't do you any good because your original changes are in the repo you pushed to.

And, yes, I removed the images on purpose since they didn't seem to convey any useful information.

**Comment by ferventcoder** [comment]
> I'm a Git newbie, though. Can I modify history that's been pushed?

Git is awesome. 👍 

Anyway, you pushed to your remote, not to the main remote (this repo). That means you can do interactive rebases and all kinds of fun things. And you can push back to your remote overwriting those changes allowing for a conversation / review to continue on GitHub. It's pretty swank. 

I'll go ahead and grab this one. We'll catch it next time when it's code changes.

**Comment by ferventcoder** [comment]
To give you a rundown (if you are interested):

* `git fetch upstream pull/19/head:pr19` (19 is the id of this pull request)
* `git checkout pr19`
image

* `git rebase master` (deletes the two merge commits and replays the commits after the master's HEAD commit)
image
* `git rebase -i HEAD~3`
image
* squash the others (change the commits to `s`/`squash`). Save file and close.
* edit the final commit message. Save and close.
image
image


Just a note before I push this though... I saw your email address as one possibly from work. Would you like me to adjust that for you? You can hit me up directly on Gitter.

**Comment by ferventcoder** [comment]
@splatteredbits Just a note before I push this though... I saw your email address as one possibly from work. Would you like me to adjust that for you? You can hit me up directly on Gitter.

**Comment by ferventcoder** [comment]
A note to clarify above - you don't ever force push to the main remote. Only to forked repositories.

**Comment by ferventcoder** [comment]
Have not forgotten about this, just FYI.

**Comment by ferventcoder** [comment]
Merged in with the correct user/email in the commit.

**Comment by ferventcoder** [comment]
https://github.com/chocolatey/choco-wiki/commit/7c21400ab7aac7763fc5061ac1a02b998e56b0ab

**Pull request closed without merging**